### ISSUE TITLE
dataset: AnnoMI

### ISSUE THREAD

**Issue by kirawi** [LAION-AI/Open-Assistant#2038]
https://github.com/uccollab/AnnoMI

According to its description, this is:

> he first publicly and freely accessible dataset of professionally transcribed dialogues demonstrating high- and low-quality motivational interviewing (MI), an effective counselling technique, with annotations on key MI aspects by domain experts.

The full dataset is expected to be released by March 15th.

**Comment by CloseChoice** [comment]
I would like to work on this.

**Comment by camsdixon1** [comment]
@CloseChoice  have you made any progress on this?

**Comment by CloseChoice** [comment]
@camsdixon1 yes, just need to upload the dataset to huggingface. I somehow forgot that.
